dali/test/python/operator_2/test_random_choice.py

```diff
@@ -0,0 +1,201 @@
+# Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
The following tests are worth adding in my opinion:
- Some probabilities equal to 0
- Only 1 element to choose from
- Randomly choosing a number for some huuge range, like [0, 2^40). This is a bit of misuse, but maybe we should check that DALI won't explode in such case

Reply:
Done. It explodes only a bit.